internal/agent/config/config.go

```diff
@@ -0,0 +1,27 @@
+package config
+
+import (
+	"flag"
+
+	"github.com/caarlos0/env/v6"
+)
+
+type Config struct {
+	ServerAddr     string `env:"ADDRESS"`
+	ReportInterval int64  `env:"REPORT_INTERVAL"`
+	PollInterval   int64  `env:"POLL_INTERVAL"`
+}
+
+func MustLoad() *Config {
```

Comment:
```suggestion
func MustLoad() (*Config, error) {
```
Все ошибки надо поднимать максимально вверх и там и обрабатывать.